bundle/src/bundler.rs

```diff
@@ -1,9 +1,13 @@
 use async_compression::futures::bufread::ZstdDecoder;
 use async_std::{io::ReadExt, stream::StreamExt};
 use async_tar_wasm::Archive;
+use context::bazel_bep::parser::BazelBepParser;
 use futures_io::AsyncBufRead;
-use std::io::{Seek, Write};
 use std::path::PathBuf;
+use std::{
+    fs::File,
+    io::{Seek, Write},
+};
```

Comment:
would be awesome if we could combine these `std` imports and stick them in their own group above these normal crate imports

Reply:
👍 Need a linter for this 🙄